py3status/modules/volume_status.py

```diff
@@ -118,7 +118,7 @@ def setup(self, parent):
             self.device = 'default'
         self.cmd = ['amixer', '-q', '-D', self.device,
                     '-c', self.card, 'sset', self.channel]
-        self.get_volume_cmd = ['amixer', '-D', self.device,
+        self.get_volume_cmd = ['amixer -M', '-D', self.device,
                                '-c', self.card, 'sget', self.channel]
```

Comment:
Hey thanks.  Is that a style convention?  No worries.

Reply:
Consistency purpose. Elements in the list here will be joined by a space. It is unlikely that the space will be changed to something else, but if that did happen, that one element would have to be fixed. It's an unlikely scenario here, but it can matter somewhere else. Just dotting the i's and crossing the t's. Thanks for keeping the list clean.

Reply:
>  Is that a style convention?

No it is because ultimately we use `subprocess.POpen()` and this need the command split into it's parts.  As a convenience we allow `Py3.command_output()` to take a string (which we split) or a list of parts.  But not a combination.

Reply:
Sorry that may not be correct because the code was vandalised at some point :(  however those changes should be reverted - I am going to go home and cry for the rest of the week